ldap/servers/slapd/bind.c

```diff
@@ -349,9 +349,9 @@ do_bind(Slapi_PBlock *pb)
             while (1) {
                 if (*pmech == NULL) {
                     /*
-                 * As we call the safe function, we receive a strdup'd saslmechanisms
-                 * charray. Therefore, we need to remove it instead of NULLing it
-                 */
+                     * As we call the safe function, we receive a strdup'd saslmechanisms
```

Comment:
Should we run clang-format again?

Reply:
We should, but not in this PR :-)

Reply:
👍

Reply:
@Firstyear have I completed your requested changes?